Analytics/Controllers/AnalyticsTreeController.cs

```diff
@@ -96,8 +98,13 @@ protected override Umbraco.Web.Models.Trees.TreeNodeCollection GetTreeNodes(stri
         /// <returns></returns>
         protected override Umbraco.Web.Models.Trees.MenuItemCollection GetMenuForNode(string id, System.Net.Http.Formatting.FormDataCollection queryStrings)
         {
-            return null;
+            var menu = new MenuItemCollection();
+            if (id == Constants.System.Root.ToInvariantString()) 
+            {
+                // root actions
+                menu.Items.Add<RefreshNode, ActionRefresh>(ui.Text("actions", ActionRefresh.Instance.Alias), false);
+            }
+            return menu;
```

Comment:
I have added a refresh menu action at root, so you can refresh the trees and then the e-commerce tree is removed if you don't have e-commerce data in Google Analytics. I would like to use navigationService.syncTree after click on save in settings, but it seems to have some issues. See the other comment about the Settings.Controller.js